### DISCUSSION TITLE
How to deploy customization forks

### DISCUSSION THREAD

**Discussion by palewire** [nytimes/library#263] [Q&A/Help] [answered]
I've successfully deployed the standard app to Heroku. Everything is a-okay. Now I want to begin to customize the vanilla deploy.

When I first created the Heroku app with the magic button, I input a `CUSTOMIZATION_GIT_REPO` variable that points to `git@github.com:datadesk/library-customization.git`, where I have forked the example customization library.

My hope is that I can simply push changes to my fork and somehow trigger them to build on Heroku, but it's unclear to me from the docs how to make this happen. Anyone have advice?

**Comment by isaacwhite** [comment, marked as answer]
Hey @palewire, we still deploy our customized Heroku sites manually using the cli integration. You should be able to do that by following the [general Heroku CLI docs](https://devcenter.heroku.com/articles/heroku-cli).

You _might_ be able to use a GitHub integration hook to autodeploy your Library site, but my hunch is that the deploys in that case would be triggered by pushes to the main nytimes/library repo, which is not what you want.

I think it would be possible to flip the current [Procfile integration](https://github.com/nytimes/library/blob/master/Procfile), so that the customization repo clones in `nytimes/library` on each deploy and moves the overrides accordingly. If the customization repo had a Procfile like that, customized autodeploys would probably fit the typical Heroku integration  pattern better. Is that something you'd be interested in exploring?

**Comment by palewire** [comment]
Not to give you even more to do, but the ideal integration from my point of view would let me push to my customization repo and have it autodeploy to Heroku. That way I'd only need that slimmed down config repository to make things go.

**Comment by maxine** [comment]
@palewire great idea - want to open a new feature request issue for this enhancement?
